### PR TITLE
axiom saves for mpteq*, prove mhphf2 (+ofeqd)

### PR DESCRIPTION
**Pull request by icecream17**
The title is just the main points

commit by commit

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -48647,13 +48647,33 @@ although the definition does not require it (see ~ dfid2 for a case
 
   ${
     $d A y $.  $d B y $.  $d C y $.  $d D y $.  $d ph y $.  $d x y $.
+    mpteq12da.1 $e |- F/ x ph $.
+    mpteq12da.2 $e |- ( ph -> A = C ) $.
+    mpteq12da.3 $e |- ( ( ph /\ x e. A ) -> B = D ) $.
+    $( An equality inference for the maps-to notation.  (Contributed by Glauco
+       Siliprandi, 23-Oct-2021.)  Drop ~ ax-10 .  (Revised by SN,
```
Comment (benjub) [inline set.mm:48654]: The standard phrase is "Remove dependency on".

File: set.mm [hunk 2]
```diff
@@ -48685,16 +48715,11 @@ although the definition does not require it (see ~ dfid2 for a case
     $( An equality inference for the maps-to notation.  (Contributed by NM,
        24-Aug-2011.)  (Revised by Mario Carneiro, 16-Dec-2013.)  Drop ~ ax-10
        while shortening its proof.  (Revised by Steven Nguyen and Gino Giotto,
-       1-Dec-2023.) $)
+       1-Dec-2023.)  Drop ~ ax-12 (recovering the OLD version).  (Revised by
```
Comment (benjub) [inline set.mm:48718]: Probaly no need for the parenthetical comment about the OLD version. Actually, you may even merge the latest two revision comments into a single "Remove dependency on ~ ax-10 , ~ ax-12 ."

File: set.mm [hunk 3]
```diff
@@ -650794,6 +650865,18 @@ D Fn ( I ... ( M - 1 ) ) /\
       ( wcel cop cxp opelxpi mp2an ) ACGBDGABHCDIGEFABCDJK $.
   $}
 
+  ${
+    $d ph f g x $.  $d R f g x $.  $d S f g x $.
+    ofeqd.1 $e |- ( ph -> R = S ) $.
+    $( Equality theorem for function operation, deduction form.  EDITORIAL:
+       Using this to prove ~ ofeq is shorter than the other way around.
+       (Contributed by SN, 11-Nov-2024.) $)
+    ofeqd $p |- ( ph -> oF R = oF S ) $=
```
Comment (benjub) [inline set.mm:650874]: Then I'm in favor of moving it to Main and using it to prove ofeq and also minimize with it theorems using ofeq.

File: set.mm (hunk 1 above)
Comment (icecream17) [reply set.mm:48654]: Done

though note currently "Avoid" is a little more common ("Remove dependenc" = 73, "Avoid ~ ax" = 103)

**Comment by icecream17** [comment]
two more opportunistic shortenings I just noticed

**Comment by icecream17** [comment]
~~based on the errors, somehow set.mm verifies but iset.mm doesn't~~ (resolved)

**Review by benjub — APPROVED** [review]

**Pull request merged**